content/integrations/postfix.md

```diff
@@ -26,13 +26,7 @@ Get metrics from Postfix in real time to monitor the messages pending in your Po
         # command to run the postfix check.  Here's an example:
         #
         # example /etc/sudoers entry:
-        #          dd-agent ALL=(ALL) NOPASSWD:/usr/bin/find
-
-        # The user running dd-agent must have passwordless sudo access for the find
-        # command to run the postfix check.  Here's an example:
-        #
-        # example /etc/sudoers entry:
-        #          dd-agent ALL=(ALL) NOPASSWD:/usr/bin/find
+        #          dd-agent ALL=(postfix) NOPASSWD:/usr/bin/find /var/spool/postfix/incoming -type f
```

Comment:
Should we stay consistent with the postfix readme from here: https://github.com/DataDog/integrations-core/blame/master/postfix/README.md#L37

```dd-agent ALL=(postfix) NOPASSWD:/usr/bin/find /var/spool/postfix/incoming -type f
dd-agent ALL=(postfix) NOPASSWD:/usr/bin/find /var/spool/postfix/active -type f
dd-agent ALL=(postfix) NOPASSWD:/usr/bin/find /var/spool/postfix/deferred -type f
```

Reply:
I think this bit of docs is in 3 places, I missed that one

Reply:
+1 on staying consistent with the postfix readme.